internal/kafka/internal/clusters/standalone_provider.go

```diff
@@ -64,6 +64,10 @@ func newStandaloneProvider(connectionFactory *db.ConnectionFactory, dataplaneClu
 // blank assignment to verify that StandaloneProvider implements Provider
 var _ Provider = &StandaloneProvider{}
 
+func (s *StandaloneProvider) RemoveResources(clusterSpec *types.ClusterSpec, syncSetName string) error {
+	return nil
```

Comment:
I'm going to add an implementation for the standalone provider as well, although I believe it's sufficient to verify the ocm provider implementation.

Reply:
I wonder how that's going to be implemented as we do not have the syncset name for standalone clusters.

If this was taking a list of resources, we could have used that somehow to issue a delete call for every of the resources: but I can't imagine the complexity involved. 

I am wondering if a NOOP is the best we can do here.

Reply:
It's not using a SyncSet in standalone mode, but installs some resources manually through the Kube API. I have to check if the Observability resources are even installed.

Reply:
> It's not using a SyncSet in standalone mode, but installs some resources manually through the Kube API

Yes. 

> I have to check if the Observability resources are even installed.

Yeap, they'll be installed. But it's not only them that can get installed, various other resources e.g operators pull secrets also gets passed down via this way.

Reply:
Another comment about this: I think there is a bit of a naming problem here:
ApplyResources creates a set of 0..N resources putting them in a given ResourceSet name. Notice how `ResourceSet` is a type that we have created in KFM.

In this PR we have created a method named RemoveResources but the method does not delete the resources you specify. You delete all the resources in the a given ResourceSet name. Therefore `RemoveResourceSet` would be a better name which takes a resourceSetName parameter.
If in the future we want to be able to delete specific resources within a given ResourceSet then that would be a different method too.